src/services/disclaimer.js

```diff
@@ -0,0 +1,153 @@
+goog.provide('ngeo.Disclaimer');
+
+goog.require('goog.asserts');
+goog.require('ngeo');
+goog.require('ngeo.Message');
+
+
+/**
+ * Provides methods to display any sort of messages, disclaimers, errors,
+ * etc. Requires Bootstrap library (both CSS and JS) to display the alerts
+ * properly.
+ *
+ * @constructor
+ * @extends {ngeo.Message}
+ * @ngdoc service
+ * @ngname ngeoDisclaimer
+ * @ngInject
+ */
+ngeo.Disclaimer = function() {
+
+  goog.base(this);
+
+  var container = angular.element('<div class="ngeo-disclaimer"></div>');
+  angular.element(document.body).append(container);
+
+  /**
+   * @type {angular.JQLite}
+   * @private
+   */
+  this.container_ = container;
+
+  /**
+   * Cache of messages.
+   * @type {Object.<string, angular.JQLite>}
+   * @private
+   */
+  this.messages_ = {};
+
+};
+goog.inherits(ngeo.Disclaimer, ngeo.Message);
+
+
+/**
+ * Show disclaimer message string or object or list of disclamer message
+ * strings or objects.
+ * @param {string|Array.<string>|ngeox.Message|Array.<ngeox.Message>}
+ *     object A message or list of messages as text or configuration objects.
+ * @export
+ */
+ngeo.Disclaimer.prototype.alert = function(object) {
+  this.show(object);
+};
+
+
+/**
+ * Close disclaimer message string or object or list of disclamer message
+ * strings or objects.
+ * @param {string|Array.<string>|ngeox.Message|Array.<ngeox.Message>}
+ *     object A message or list of messages as text or configuration objects.
+ * @export
+ */
+ngeo.Disclaimer.prototype.close = function(object) {
+  var msgObjects = this.getMessageObjects(object);
+  msgObjects.forEach(this.closeMessage_, this);
+};
+
+
+/**
+ * Show the message.
+ * @param {ngeox.Message} message Message.
+ * @protected
+ */
+ngeo.Disclaimer.prototype.showMessage = function(message) {
+  var type = message.type;
+  goog.asserts.assertString(type, 'Type should be set.');
+
+  var uid = this.getMessageUid_(message);
+  if (this.messages_[uid] !== undefined) {
+    return;
+  }
+
+  var classNames = ['alert', 'fade', 'alert-dismissible'];
+  switch (type) {
```

Comment:
Because it was asked in the specs, see: https://github.com/camptocamp/c2cgeoportal/wiki/Spec-%231769-Disclaimer#architecture

>Manage different colors like the notifier